### PR TITLE
Create products group

### PR DESCRIPTION
**Pull request by brentleyjones**

### CODE REVIEW

**Comment by brentleyjones** [comment]
Current dependencies on/for this PR:
* main
  * **PR #22** <a href="https://app.graphite.dev/github/pr/buildbuddy-io/rules_xcodeproj/22" target="_blank">Graphite</a> 
    * **PR #23** <a href="https://app.graphite.dev/github/pr/buildbuddy-io/rules_xcodeproj/23" target="_blank">Graphite</a>  (this PR)
      * **PR #24** <a href="https://app.graphite.dev/github/pr/buildbuddy-io/rules_xcodeproj/24" target="_blank">Graphite</a> 
        * **PR #25** <a href="https://app.graphite.dev/github/pr/buildbuddy-io/rules_xcodeproj/25" target="_blank">Graphite</a> 
          * **PR #26** <a href="https://app.graphite.dev/github/pr/buildbuddy-io/rules_xcodeproj/26" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://graphite.dev?utm_source=gh).

File: tools/generator/src/PBXProductType+Extensions.swift [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import XcodeProj
+
+extension PBXProductType {
+    var fileType: String? {
+        switch self {
+        case .application: return "wrapper.application"
```
Comment (tylerwilliams) [inline tools/generator/src/PBXProductType+Extensions.swift:6]: nit: want to add a comment linking to the source of this mapping?

**Review by tylerwilliams — APPROVED** [review]
LGTM

File: tools/generator/src/PBXProductType+Extensions.swift (hunk 1 above)
Comment (brentleyjones) [reply tools/generator/src/PBXProductType+Extensions.swift:6]: There isn't one. I created each of these targets in Xcode and looked at the product for each and what file type was associated with them.

**Pull request merged**